packages/app/src/app/pages/Dashboard/Content/routes/Settings/TeamSettings/WorkspaceSettings.tsx

```diff
@@ -463,16 +475,68 @@ export const WorkspaceSettings = () => {
           </Stack>
         )}
       </Stack>
-      {isTeamAdmin && numberOfEditorsIsOverTheLimit && (
+
+      {/**
+       * Limit free plan amount of editors.
+       */}
+      {isTeamAdmin && (numberOfEditorsIsOverTheLimit || hasMaxNumberOfEditors) && (
         <MessageStripe justify="space-between">
-          Free teams are limited to 5 editor seats. Some permissions might have
-          changed.
+          <span>
+            {numberOfEditorsIsOverTheLimit && (
+              <>
+                Free teams are limited to 5 editor seats. Some permissions might
+                have changed.
+              </>
+            )}
+            {hasMaxNumberOfEditors && (
+              <>
+                You&apos;ve reached the maximum amount of free editor seats.
+                Upgrade for more.
+              </>
+            )}
+          </span>
+          <MessageStripe.Action
+            {...(checkout.state === 'READY'
+              ? {
+                  as: 'a',
+                  href: checkout.url,
+                }
+              : {
+                  as: RouterLink,
+                  to: '/pro',
+                })}
+            onClick={() =>
+              track(
+                isEligibleForTrial
+                  ? 'Limit banner: team editors - Start Trial'
+                  : 'Limit banner: team editors - Upgrade'
+              )
+            }
+          >
+            {isEligibleForTrial ? 'Start trial' : 'Upgrade now'}
+          </MessageStripe.Action>
         </MessageStripe>
       )}
-      {isTeamAdmin && hasMaxNumberOfEditors && (
+
+      {/**
+       * Soft limit for pro teams.
+       */}
+      {numberOfEditors > MAX_PRO_EDITORS && (
         <MessageStripe justify="space-between">
-          You&apos;ve reached the maximum amount of free editor seats. Upgrade
-          for more.
+          <span>
+            You have over {MAX_PRO_EDITORS} editors. Upgrade to the Organization
```

Comment:
@lenasmayor I used `Organization` instead of `Enterprise` since it's the name used on the pricing page.